lib/jekyll-admin/server/page.rb

```diff
@@ -85,7 +84,9 @@ def relative_page_path
       end
 
       def page
-        site.pages.find { |p| sanitized_path(p.path) == page_path }
+        Jekyll::Page.new(
```

Comment:
I think he's asking if we want to look to the existing page first then fall back to creating a new one like we do for other content types?